modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+
+=== New `transparent` property for `iframe` dialog component
+
+As part of the fix for correcting the presentation of {productname} content in the *Preview* and *Insert template* dialogs, the `iframe` dialog component now supports a new `transparent` property.
+
+
+=== New `removeAttributeFilter` and `removeNodeFilter` functions added to the DomParser and DOM Serializer APIs
+
+Two new filter functions — `removeAttributeFilter` and `removeNodeFilter` — have been added to the `DomParser` and DOM `Serializer` APIs.
+
+Each function takes two arguments: `name` and `callback`.
+
+`name` is required.
+
+`callback` is optional.
+
+`name` can take a comma-separated list of attribute or node names.
+
+If both `name` and `callback` are provided, the functions remove the specified callback.
+
+If only `name` is provided, the functions remove all callbacks for the specified name.
+
+When only `name` is provided, the new functions work in equivalent fashion to the `editor.off` API, which removes all event handlers when only a name is provided.
+
+//// An example would be a good thing. Also for the updated docs proper.
+
+
+=== New `iframe_template_callback` option in the media plugin.
+
+The new `iframe_template_callback` option, provided by https://github.com/Namstel[Namstel], is an addition to the media plugin.
+
+The `iframe_template_callback` option takes a function and replaces the default callback when the source URL matches a valid iframe.
+
+It works equivalently to the `audio_template_callback` and `video_template_callback` options, but for iframes.
+
+
+=== New `dispatchChange` UndoManager API to fire the change with current editor status as `level` and current undoManager layer as `lastLevel`
+
+The new `dispatchChange` API allows for the generating of an undo level outside the current editor’s context.
+
+It takes the current editor undo level status as `level` and the current `undoManager` layer as `lastLevel`. When a change event fires, however, `dispatchChange` does not put the change in undo level on to the `undoManager` stack.
+
+// An example would be a good thing. Also for the updated docs proper.
+
+
+
+[[known-issues]]
+== Known issues
+
+This section describes issues that users of {productname} 6.1 may encounter and possible workarounds for these issues.
+
+There are three known issues affecting {productname} 6.1.
+
+=== Clearer focus states for buttons while navigating with keyboard does not work in older browsers
+
+{productname} 6.1 introduced a new usability feature: a blue outline that more clearly highlights the currently active button when navigating {productname} dialogs with a keyboard.
+
+This feature relies on the `:focus-visible` pseudo-selector.
+
+This pseudo-selector is supported in current and recent versions of all major browsers, but it is not supported in older browser versions.
```

Comment:
This is incorrect... it doesn't work in recent Safari versions, it only works in the latest Safari version. If it worked in recent versions then we wouldn't need this known issue.
```suggestion
This pseudo-selector is supported in current versions of all major browsers, but it is not supported in older browser versions.
```